tests/components/logger/test_websocket_api.py

```diff
@@ -124,6 +124,7 @@ async def test_custom_integration_log_level(
         assert msg["success"]
 
         assert async_get_domain_config(hass).overrides == {
+            "homeassistant.components.foobar": logging.DEBUG,
```

Comment:
I think this is a remnant of me trying to figure out what I wasn't patching correctly.

I think we can drop this again and change L115 to "hue" again for consistency.

Reply:
Its still coming in as foobar because thats what the mocking mocks it as

Reply:
Yeah, I'll test a clean up locally and push it up in a sec

Reply:
57cadfe8d17cdeacf4bf5aa5bc77ac1dede983fb

Reply:
Ah, you did it already! Thanks